src/compile/selection/transforms/zoom.ts

```diff
@@ -49,17 +54,6 @@ const zoom:TransformCompiler = {
       onDelta(model, selCmpt, 'y', 'height', signals);
     }
 
-    const size = signals.filter((s:any) => s.name === name + INTERVAL_SIZE);
```

Comment:
The signal has been removed as part of breaking visual/pixel and data signals apart.